src/Compilers/Core/Portable/WellKnownMember.cs

```diff
@@ -499,7 +499,7 @@ internal enum WellKnownMember
         System_Threading_CancellationTokenSource__CreateLinkedTokenSource,
         System_Threading_CancellationTokenSource__Token,
         System_Threading_CancellationTokenSource__Dispose,
-
+        System_ArgumentNullException__ctor,
```

Comment:
consider naming this `System_ArgumentNullException__ctorString` to indicate that it is the constructor that takes a `String` and not the no-arg constructor